arch/src/x86_64/mod.rs

```diff
@@ -363,10 +318,82 @@ impl<const FUNCTION: u32, const INDEX: u32, const REG: u8> CpuIdFeatureFlags<FUN
                     REG
                 );
             }
+        } else {
+            // TODO: Better log or return an error here
+            log::warn!("no entry matched");
         }
     }
 }
 
+/// Reduces boilerplate when implementing CpuIdFeatureFlag constants.
```

Comment:
Macro to generate the boilerplate when implementing [`CpuIdFeatureFlag`] constants